src/Views/Error/Index.cshtml

```diff
@@ -1,6 +1,7 @@
 ﻿@{
-    ViewData["Title"] = "Error";  
-    Layout = "_ds_ErrorLayout";
+    ViewData["Title"] = "Error";
+    Layout = "_ds_ErrorLayout";
+    ViewData["BaseUrl"] = $"../{Model.BaseURL} ";
```

Comment:
Error view does not have a ViewModel, form errors due to null binding on Error view, formTitle not required on error view.